### ISSUE TITLE
New NPC Sheet

### ISSUE THREAD

**Issue by Galdormin** [justinross/foundryvtt-blades-in-the-dark#7]
What do you think about redesigning the NPC sheets? 
I thought of a similar model with the information from the existing sheet and the People generator from the Core Book.

![NPC-1](https://user-images.githubusercontent.com/76685550/131373572-371feb0b-9847-4850-bdc7-46e0e49cd2bb.png)

With an example with Bazso Baz :

![NPC-2](https://user-images.githubusercontent.com/76685550/131373641-c2b10b3f-69b2-446a-9b10-ab8d614796cf.png)

**Comment by justinross** [comment]
Nice! My goal was always kind of to redesign all of the sheets to match the new character sheet. I'm hoping I can just copy most of the PC sheet over for use in the NPC sheets, too.

I'll need to add a bunch of fields to the NPC data template to handle all of the data you've included in your mockup (which, I think, makes it harder to release as a module instead of an update to the system), but I think it'd be nice to have codified space for all of that (rather than just shoving it into a notes text field).

**Comment by Galdormin** [comment]
`I'll need to add a bunch of fields to the NPC data template to handle all of the data you've included in your mockup`

Yes, and especially since these are fields that can be used to create random NPCs with a macro.

**Comment by justinross** [comment]
Yeah, that'd be nice. Adding the fields is easy enough, and doesn't break anything. Shouldn't be a problem. :)
